.github/workflows/smoke.yml

```diff
@@ -16,6 +16,7 @@ concurrency:
 
 env:
   GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  SMOKE_TEST_BRANCH: main
```

Comment:
nice! Making life simple down the road.

Reply:
I've had to re-figure this out too many times.